### PR TITLE
Do not use Svgs on the main screen and hooks

### PR DESCRIPTION
**Pull request by osdnk**

### CODE REVIEW

**Review by terrysahaidak — DISMISSED** [review]
Overall looks good. I would implement "shouldComponentUpdate"  to the SvgImage component to avoid any rerenders.

**Comment by skylarbarrera** [comment]
new things:  
- I imagine we want to have a border radius to match on the expanded state: https://cloud.skylarbarrera.com/Screen-Shot-2021-11-18-14-16-40.26.png
- persistentDominantColors are off -  doesn't seem like we're grabbing from the svgToPng url when we convert it, I wonder if we should add that logic into the image or card components so we have it up front when opening the expanded state: https://cloud.skylarbarrera.com/Screen-Shot-2021-11-18-14-17-03.97.png

File: package.json [hunk 1]
```diff
@@ -109,6 +109,7 @@
     "browserify-zlib": "0.1.4",
     "buffer": "4.9.2",
     "chroma-js": "2.1.0",
+    "cloudinary": "^1.27.1",
```
Comment (brunobar79) [inline package.json:112]: Exact match!

File: src/handlers/svgs.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { image as cloudinaryImage } from 'cloudinary/lib/cloudinary';
+import cloudinaryConfig from 'cloudinary/lib/config';
+import {
+  // @ts-ignore
+  CLOUDINARY_API_KEY as apiKey,
+  // @ts-ignore
+  CLOUDINARY_API_SECRET as apiSecret,
+  // @ts-ignore
+  CLOUDINARY_CLOUD_NAME as cloudName,
```
Comment (brunobar79) [inline src/handlers/svgs.js:9]: You need to PR this ones to .env repo and also add them in CI

**Review by brunobar79 — CHANGES_REQUESTED** [review]
all good except deps!

**Review by brunobar79 — DISMISSED** [review]
LGTM!

**Review by brunobar79 — DISMISSED** [review]

**Comment by ibrahimtaveras00** [comment]
As @skylarbarrera has mentioned above, there are some groups that don't show up at all such as The Colors (thecolors.art) on skillet.eth 

Screen Shot 2021-11-23 at 1 16 59 PM

In some instances, if I kill the app and relaunch sometimes it does load but only for certain groups.

If we're ok with this for this PR I'd say we're good to pass.  WDYT @osdnk?

**Review by ibrahimtaveras00 — APPROVED** [review]

**Pull request merged**